<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>1100</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>1100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>1100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:grey</string>
   </property>
   <widget class="QPushButton" name="skip">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>1000</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color:rgb(142, 237, 255);
border-radius: 20px;
font: 12pt &quot;MS Shell Dlg 2&quot;;
</string>
    </property>
    <property name="text">
     <string>Next Question</string>
    </property>
   </widget>
   <widget class="QPushButton" name="home">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>150</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: white;
border: none;

</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/newPrefix/images/arrow.png</normaloff>:/newPrefix/images/arrow.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="input">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(79, 79, 79);
color: white;
font: 16pt &quot;MS Shell Dlg 2&quot;;
border-radius:12px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="output">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>370</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(79, 79, 79);
color: white;
font: 16pt &quot;MS Shell Dlg 2&quot;;
border-radius:12px
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>700</width>
      <height>209</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font: 27pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Hard Mode</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="questions_label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>380</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font: 14pt &quot;MS Shell Dlg 2&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="message">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>990</y>
      <width>300</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="go">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>880</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color:rgb(255, 152, 142);
border-radius: 15px;
</string>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
